topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.md

```diff
@@ -13,12 +13,16 @@ key_points:
   - "Generating variant protein database"
   - "Generating genomic and variant mapping files for visualization"
 follow_up_training:
-  -
-    type: "internal"
+  - type: "internal"
     topic_name: proteomics
     tutorials:
-      - proteogenomics-novel-peptide-analysis
+      - proteogenomics-dbsearch
 
+  - type: "internal"
+    topic_name: proteomics
+    tutorials:
+      - proteogenomics-novel-peptide-analysis
```

Comment:
you can move this line to the above, since both tutorials are from the same topic, so:

```
follow_up_training:
  - type: "internal"
    topic_name: proteomics
    tutorials:
      - proteogenomics-dbsearch
      - proteogenomics-novel-peptide-analysis
```

Reply:
thanks, made that change